server/src/modules/mentorship-request/mentorship-request.model.ts

```diff
@@ -36,10 +36,10 @@ export class MentorshipRequest extends Model<MentorshipRequest> {
   requestMessage: string;
 
   @Column({ type: DataType.DATE, allowNull: false })
-  startDate: Date;
+  startDate: string;
```

Comment:
In a perfect world it'd be great to keep `Date` here and we could have functions to convert `MentorshipRequest` to `MentorshipRequestDTO`, but if this is simpler then no pressure.